cdc/redo/manager.go

```diff
@@ -302,7 +305,11 @@ func (m *ManagerImpl) RemoveTable(tableID model.TableID) {
 	} else {
 		log.Warn("remove a table not maintained in redo log manager", zap.Int64("table-id", tableID))
 	}
-	// TODO: send remove table command to redo log writer
```

Comment:
Redo log applier will sort redo logs, so the writer doesn't need to care about table scheduling.